packages/integration-test/tests/index.test.ts

```diff
@@ -77,12 +76,12 @@ process.env.BLOCK_CONFIRMATIONS = '0'
 jestExtendToCompareBigNumber(expect)
 
 describe('testnet', () => {
-  // attachConsoleLogToPino()
   let context!: Context
   const ctx = () => context
   beforeAll(async () => {
-    if (process.env.DEBUG) attachConsoleLogToPino()
-    attachConsoleErrorToPino()
+    if (process.env.DEBUG === 'true') {
```

Comment:
I've got this reverted in my branch. I much prefer the truthy check so that `DEBUG=1` enables it.

Reply:
reverted back this ;)